chain/rpc/src/client.rs

```diff
@@ -1270,7 +1270,7 @@ mod tests {
 
     #[test_log::test(async_std::test)]
     async fn test_client_from_file() -> anyhow::Result<()> {
-        let block_time = Duration::from_secs(1);
+        let block_time = Duration::from_millis(1100);
```

Comment:
Anvil has second precision, so this will likely have no effect.

Reply:
That is true for anvil, but the block time is also used in the following loop for sleeping, which is where a race-conditions occurs if its set to the exact block time used by Anvil.